alexandria/core/tests/test_views.py

```diff
@@ -196,23 +199,26 @@ def test_at_rest_encryption(admin_client, settings, document, mocker):
 def test_validate_created_by_group(
     db,
     settings,
+    attribute,
     viewset,
     request,
     update,
     admin_client,
+    admin_user,
+    admin_groups,
     active_group,
```

Comment:
`active_group` is somewhat misleading after parametrizing the `group` and `user` attribute

rename to `active_attribute`?

Reply:
Makes sense done